docs/BR.md

```diff
@@ -298,6 +298,8 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **CA Key Pair**: A Key Pair where the Public Key appears as the Subject Public Key Info in one or more Root CA Certificate(s) and/or Subordinate CA Certificate(s).
 
+**Canonical Authorization Domain Name**: The domain name found within the RDATA value of a CNAME record located at an FQDN composed of an Authorization Domain Name that is prefixed with a Domain Label that begins with an underscore character.
```

Comment:
I'm not an english native speaker and struggle to grasp this definition. Could we maybe add an example to make it clearer? Especially the reference to the Domain Label confuses me... :(

Reply:
```suggestion
**Canonical Authorization Domain Name**: The domain name found within the RDATA value of a CNAME record located at an Authorization Domain Name that is prefixed with a Domain Label that begins with an underscore character.
```
@romanf Would this minor change make the definition any clearer? 

The goal is to define `accountbindingid.cadomain.com` as the Canonical Authorization Domain Name of the CNAME record: `_somethingsomething.example.com. IN CNAME accountbindingid.cadomain.com`

The domain: `_somethingsomething.example.com` part is an underscore prefixed Authorization Domain Name which is currently referenced in section 3.2.2.4.7.

> Confirming the Applicant's control over the FQDN by confirming the presence of a Random Value or Request Token for either in a DNS CNAME, TXT or CAA record for either 1) an Authorization Domain Name; or 2) **an Authorization Domain Name that is prefixed with a Domain Label that begins with an underscore character**.

Reply:
I like the example and I think it should be added to the BRs to assist in better understanding the requirement.

Reply:
I'm still a bit confused, sorry. :(
So, 
1. _somethingsomething would be either defined by the applicant or the CA to validate ownership of "example.com" (and *.example.com?). The regulation would not specify who has to specify what's after the _, correct?

2. The CNAME then points to an entry in a CA-controlled DNS domain (cadomain.com). Whereby the CA defines the "accountbindingid". Would this be per applicant or per domain that is to be validated?

3. The type of that entry in the CA-controlled DNS domain could be (according to 3.2.2.4.7) CNAME, TXT or CAA (where CNAME probably doesn't make any sense?). Correct?

4. The CA would then put the random value that validates "example.com" in that entry.

Hope I got it right this time. :)
Thx
Roman

Reply:
Hi @romanf,
>     1. _somethingsomething would be either defined by the applicant or the CA to validate ownership of "example.com" (and *.example.com?). The regulation would not specify who has to specify what's after the _, correct?

Correct, the underscore-prefixed subdomain name is not security critical so it doesn't matter who defines it. This aligns with the current method 7, which does not specify how the underscore-prefixed subdomain name is created.

>     2. The CNAME then points to an entry in a CA-controlled DNS domain (cadomain.com). Whereby the CA defines the "accountbindingid". Would this be per applicant or per domain that is to be validated?

It must be per Applicant, otherwise it could be possible for attackers to request certificates for domains that they do not control. It's acceptable for a CA to create a unique per-Applicant, per-domain "accountbindingid", but the security critical component is the per-Applicant uniqueness.

>     3. The type of that entry in the CA-controlled DNS domain could be (according to 3.2.2.4.7) CNAME, TXT or CAA (where CNAME probably doesn't make any sense?). Correct?

I imagine the common case will be TXT, but the other record types that you mentioned are also fine.
 
>     4. The CA would then put the random value that validates "example.com" in that entry.

Yes.

Hopefully this helps. If you think the language can be improved, any suggestions you have would be appreciated.

Reply:
@romanf - Does the explanation @CBonnell provided make sense to you? Do you have any suggested improvements?

Reply:
Yes, sorry, should have given feedback.

@CBonnell Thanks for the confirmations and explanations!

I'm not a DNS expert, so I'm just trying to understand the meaning and scope of this change and don't have any suggestions for improvements.